vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/AbstractNoW3c.java

```diff
@@ -32,22 +36,36 @@
 
     @Override
     public void setup() throws Exception {
-        if (getRunOnHub(getClass()) != null
-                        || Parameters.getHubHostname() != null) {
-
-            ChromeOptions options = new ChromeOptions();
-            options.addArguments(
-                    new String[] { "--headless", "--disable-gpu" });
-            options.setExperimentalOption("w3c", false);
-
-            options.merge(getDesiredCapabilities());
+        final WebDriver webDriver = createChromeDriverWithoutW3c(
+            getLocalExecution(), getDesiredCapabilities(), getHubURL());
+        if (webDriver != null) {
             setDesiredCapabilities(getDesiredCapabilities());
-
-            WebDriver driver = TestBench.createDriver(
-                    new RemoteWebDriver(new URL(getHubURL()), options));
-            setDriver(driver);
+            setDriver(webDriver);
         } else {
             super.setup();
         }
     }
+
+    public static WebDriver createChromeDriverWithoutW3c(
+        Optional<LocalExecution> localExecution, DesiredCapabilities capabilities, String hubURL) throws Exception {
+
+        final ChromeOptions options = createChromeOptions();
+        options.merge(capabilities);
+
+        if(!localExecution.isPresent()) {
+            return TestBench.createDriver(
+                new RemoteWebDriver(new URL(hubURL), options));
+        }else if(localExecution.get().value() == Browser.CHROME) {
```

Comment:
`if(` -> `if (`  
`}else if(` -> `} else if (`

Reply:
Thanks for the find. Fixed.